<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModelCheckboxWidget</class>
 <widget class="QWidget" name="ModelCheckboxWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>28</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>28</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>28</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Model Checkbox</string>
  </property>
  <layout class="QHBoxLayout" name="mainLayout">
   <property name="spacing">
    <number>8</number>
   </property>
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QCheckBox" name="checkboxModel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelModelName">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>600</weight>
      </font>
     </property>
     <property name="text">
      <string>Model Name</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    color: #333;
    padding: 2px 4px;
}</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelProvider">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>18</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>18</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Provider</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    background-color: #f0f0f0;
    color: #555;
    border: 1px solid #ddd;
    border-radius: 9px;
    padding: 1px 4px;
    font-weight: 500;
}</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelCapabilities">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>18</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>18</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>7</pointsize>
      </font>
     </property>
     <property name="text">
      <string>capabilities</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    background-color: #e8f4fd;
    color: #1976D2;
    border: 1px solid #90caf9;
    border-radius: 9px;
    padding: 1px 4px;
    font-weight: 500;
}</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>